deploy.js

```diff
@@ -31,7 +31,9 @@ async function deployContract() {
   const { contractTxId, srcTxId } = deployResult;
   console.log(`Deployment completed.\nDeployer: ${walletAddress}\nContract address: ${contractTxId}\nCode address: ${srcTxId}`);
 
-  return deployResult;
+  // Displaying the deployed contract code
+  const contractCode = await arweave.transactions.getData(srcTxId, { decode: true, string: true });
```

Comment:
You don't have to re-download code... What I meant is showing Transaction ID of code that you've uploaded to arweave